src/core/load-fragment.c

```diff
@@ -4629,6 +4632,49 @@ int config_parse_load_credential(
                 }
         }
 
+        r = stat(q, &st);
```

Comment:
please do not actually access stuff when loading configuration. loading configuraiton should just parse stuff, but the actually fs accesses should happen when we need to load the credentials, not earlier.